salt/modules/gpg.py

```diff
@@ -516,6 +532,11 @@ def delete_key(
     gnupghome
         Specify the location where GPG keyring and related files are stored.
 
+    use_passphrase
+        Whether to use a passphrase with the signing key. Passphrase is received
+        from Pillar.
+        
+        .. versionadded:: 3003
```

Comment:
```suggestion
    use_passphrase
    
        .. versionadded:: 3003
        
        Whether to use a passphrase with the signing key. Passphrase is received
        from Pillar.
```

Reply:
The initial problem were spaces in the empty line before `.. versionadded`, so it fails with trailing spaces errors. Fixed now.